jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/TimelinesTests.java

```diff
@@ -72,9 +75,11 @@ public void colorTimeLineDataTest(int index, String time, String event, String c
     @Test
     public void denseTimeLineTest() {
         denseTimeLine.has().size(1);
+        denseTimeLine.is().dense();
         TimeLineItem<IconAlert, UIElement> singleItem = denseTimeLine.item(1);
         singleItem.body().icon().is().type("mdi-information");
         singleItem.body().has().text(LOREM_IPSUM_TEXT);
+        singleItem.body().has().css("color", WHITE.value());
         singleItem.body().has().css("background-color", BLUE.value());
```

Comment:
Please create method "backgroundColor" to check background-color
And add the same for all color checks